src/repositories/projectRepository.ts

```diff
@@ -17,6 +21,81 @@ export const findProjectBySlug = (
     .getOne();
 };
 
+export const verifyMultipleProjects = async (params: {
+  verified: boolean;
+  projectsIds: string[];
+}): Promise<UpdateResult> => {
+  return Project.createQueryBuilder('project')
+    .update<Project>(Project, { verified: params.verified })
+    .where('project.id IN (:...ids)')
+    .setParameter('ids', params.projectsIds)
+    .returning('*')
+    .updateEntity(true)
+    .execute();
+};
+
+export const updateProjectWithVerificationForm = async (
+  verificationForm: ProjectVerificationForm,
+  project: Project,
+): Promise<Project> => {
+  const projectUpdate = await ProjectUpdate.findOne({
+    isMain: true,
+    projectId: project.id,
+  });
+
+  if (!projectUpdate) throw new Error(errorMessages.PROJECT_UPDATE_NOT_FOUND);
+
+  projectUpdate.mission = verificationForm.milestones.mission!;
+  projectUpdate.achievedMilestones =
+    verificationForm.milestones.achievedMilestones!;
+  projectUpdate.foundationDate = new Date(
+    Date.parse(String(verificationForm.milestones.foundationDate)),
+  );
+  projectUpdate.managingFundDescription =
+    verificationForm.managingFunds.description!;
+  projectUpdate.isNonProfitOrganization =
+    verificationForm.projectRegistry.isNonProfitOrganization!;
+  projectUpdate.organizationCountry =
+    verificationForm.projectRegistry.organizationCountry!;
+  projectUpdate.organizationWebsite =
+    verificationForm.projectRegistry.organizationWebsite!;
+  projectUpdate.organizationDescription =
+    verificationForm.projectRegistry.organizationDescription!;
+
+  await projectUpdate.save();
+
+  // TODO: update links in project update. Or is it enough with the project one to many?
+  const relatedAddresses: RelatedAddress[] = [];
+
+  for (const relatedAddress of verificationForm.managingFunds
+    .relatedAddresses) {
+    relatedAddresses.push(
+      RelatedAddress.create({
+        title: relatedAddress.title,
+        address: relatedAddress.address,
+        networkId: relatedAddress.networkId,
+        projectId: verificationForm.projectId,
+        userId: verificationForm.userId,
+        isPrimaryAddress: false,
+      }),
+    );
+  }
+  project.relatedAddresses = relatedAddresses;
+  return project.save();
```

Comment:
@CarlosQ96  Why you dont copy the data to project, and you are just copying that in projectUpdate?

Reply:
Hey Mohammad, I just looked at Milestone and the attributes you added them to the projectupdate. Should I add them too to the project? I feel like most info is related to a project update.